capa/features/extractors/dnfile/extractor.py

```diff
@@ -89,7 +91,9 @@ def extract_global_features(self):
         yield from self.global_features
 
     def extract_file_features(self):
-        yield from capa.features.extractors.dnfile.file.extract_features(self.pe)
+        yield from capa.features.extractors.dnfile.file.extract_features(
+            file_ctx={"pe": self.pe, "min_len": self.min_len}
```

Comment:
```suggestion
            ctx={"pe": self.pe, "min_str_len": self.min_str_len}
```

Reply:
given that we're invoking the "file" extractor, its not necessary to include the term `file_` in the parameter name.

Reply:
while this isn't wrong, I'd just like to keep the code as concise and consistent as possible.